src/components/Truncate/Truncate.tsx

```diff
@@ -70,24 +75,25 @@ export class Truncate extends React.PureComponent<
       /* istanbul ignore next */
       if (!this.node || !this.contentNode) return false
 
-      // 1. Normalizes the display to allow for calculation
-      // TODO: fix typescript complains
-      // @ts-ignore
-      this.contentNode.style.display = 'initial'
-      // 2. Calculate the differences
       const isContentTruncated =
-        // TODO: fix typescript complains
-        // @ts-ignore
-        this.contentNode.offsetWidth > this.node.offsetWidth
-      // 3. Resets the display
-      // TODO: fix typescript complains
-      // @ts-ignore
-      this.contentNode.style.display = null
```

Comment:
Removing line 76 and 85 haven't seemed to have had an effect, am I missing some use case where they might be crucial?